.env

```diff
@@ -0,0 +1,31 @@
+TYPE="source" # OR "channel"
+
+# Used when TYPE==channel
+ALGOD_CHANNEL="nightly"
+
+# Used when TYPE==source
+ALGOD_URL="https://github.com/algorand/go-algorand"
+ALGOD_BRANCH="master"
+ALGOD_SHA=""
+
+# Used regardless of TYPE
+NETWORK_TEMPLATE="images/algod/DevModeNetwork.json"
```

Comment:
Since we use the sandbox network templates, it looks like we could delete the `network_config` directory in this repo

Reply:
This [build](https://app.circleci.com/pipelines/github/algorand/py-algorand-sdk/805/workflows/302732f3-ee1e-4fff-866d-4c814be3abdd/jobs/3377) passes on py-sdk after the deletion you suggested, so am resolving this. Thanks!!!!

Reply:
FYI - I've added this explanatory comment:

<img width="813" alt="image" src="https://user-images.githubusercontent.com/291133/183939712-ddb0182c-9d4c-4388-9383-86f343a2eff5.png">

Reply:
This comment reminded me that `goal network create` has a new `--devMode` flag. It allows you to take an existing single-node network template, and turn on dev mode. Setting `DEV_MODE=true` instead of a magic template file might be more natural.

Reply:
I'll leave this for a future improvement. This requires some further investigation for how to deal with template file in the sandbox.

Reply:
@winder 

I've created this sandbox issue: https://github.com/algorand/sandbox/issues/139